scripts/gha/test_lab.py

```diff
@@ -0,0 +1,334 @@
+r"""Tool for sending mobile testapps to Firebase Test Lab for testing.
```

Comment:
should this r be here?

Reply:
That designates the string as a raw string, so that it ignores escape characters (such as the continuation in the example shell command). Similar to @ in C#.

Reply:
Gotcha. Thanks!